tests/data_protection/oadp/conftest.py

```diff
@@ -166,7 +168,8 @@ def rhel_vm_from_existing_dv(imported_dv_second_namespace):
             vm=vm,
             filename=FILE_NAME_FOR_BACKUP,
             content=TEXT_TO_TEST,
-            stop_vm=False,
+            console_impl=console.RHEL,
```

Comment:
This was removed recently, please check if you need this

Reply:
ack